packages/frontend/src/components/UserSettingsModal/AppearancePanel/AppearancePanel.styles.ts

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+
+export const AppearancePanelWrapper = styled.div`
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+`;
+
+export const Title = styled.div``;
+export const ColorPalette = styled.div`
+    display: flex;
+    flex-wrap: wrap;
+`;
+
+export const AppearanceColorWrapper = styled.div`
+    margin: 10px 5px;
+`;
+export const ColorLabel = styled.div`
+    background-color: white;
+    padding: 5px;
```

Comment:
We are trying to standardise the use of `ems` across the app. `Ems` is more beneficial as if at some point we decide to change the base font size for the app, then it will avoid us having to manually update every style across the site and wont break the designs.

[I like this site](http://pxtoem.com/) to calculate them, though there is a lot of tools out there for the same purpose :)